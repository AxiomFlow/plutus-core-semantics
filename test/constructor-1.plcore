(module Foo (imported) (exported () ()) (local () ())
  (define result
    (case (con SomeMod.Foo 1)
          (cl SomeMod.Foo (x)  (builtin addInt (10 x)))
          (cl SomeMod.NotThis ()  -1))))
