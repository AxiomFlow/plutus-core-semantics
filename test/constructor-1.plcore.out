<T> <k> . </k> <env> result |-> 11 </env> <subst> .Map </subst> </T>
